app/config/question-bank.js

```diff
@@ -76,13 +76,13 @@ const questionBank = {
       title: 'Eligibility',
       questions: [
         {
-          key: 'project-subject',
+          key: 'project-subject', // yikes! https://tenor.com/bCtQ7.gif
```

Comment:
can you remove the comments from here @salfedev

Reply:
done